config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  get '/events', to: 'events#index'
+  root to: 'events#index'
```

Comment:
Consider using the `root` method instead of `get '/'` to define the root route. This provides a more explicit and readable way to define the root route.